launch/dlio.launch

```diff
@@ -59,6 +59,7 @@
 
   <!-- RViz -->
   <node pkg="rviz" type="rviz" name="dlio_rviz" args="-d $(find direct_lidar_inertial_odometry)/launch/dlio.rviz" if="$(arg rviz)"/>
-
+  
+  <node pkg="direct_lidar_inertial_odometry" type="dlio_to_can" name="dlio_2_can" output="screen"/>
```

Comment:
I would like a flag, just like in the line above, so an argument is checked to decide if the script is ran.

`if="$(arg rviz)"/>`

but maybe something like:
`if="$(arg can)"/>`

Reply:
check chatgpt how to do this, i.e. if you also have to add something similar to`args="-d $(find direct_lidar_inertial_odometry)/launch/dlio.rviz"` or not. You could also check other packages in the repo